lib/dor/update_marc_record_service.rb

```diff
@@ -48,7 +48,7 @@ def write_symphony_record symphony_record
         return
       end
       symphony_file_name = "#{Dor::Config.release.symphony_path}/sdr-purl-856s"
-      command = "#{Dor::Config.release.write_marc_script} '#{symphony_record}' #{symphony_file_name}"
+      command = "#{Dor::Config.release.write_marc_script} \'#{symphony_record}\' #{symphony_file_name}"
```

Comment:
The reason behind the single quote was required by the unix to be able to send the string with space to the echo command. I'm not if this has been changed in the new implementation.